projects/canopy/src/lib/docs/principles/accessibility/overview.stories.mdx

```diff
@@ -0,0 +1,210 @@
+import { Meta } from '@storybook/addon-docs';
+import Feedback from '../../common/feedback.md';
+
+<Meta title="Principles/Accessibility/Overview" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />
+
+# Overview
+
+> "The power of the Web is in its universality.
+> Access by everyone regardless of disability is an essential aspect."
+>
+> <cite>Tim Berners-Lee, W3C Director and inventor of the World Wide Web</cite>
+
+## Introduction
+
+Before diving into explaining some of the key aspects of web accessibility here are some interesting facts to make you understand how important it is to create websites that are accessible by any user no matter their disabilities:
+
+- there are around 14.6 million disable people in the UK (almost one in five of the population).
```

Comment:
Should this be, "there are around 14.6 million disable**d** people in the UK..."?